src/python/foglamp/sending_process.py

```diff
@@ -231,7 +223,7 @@ def _retrieve_configuration(self, stream_id):
         .. todo::
         """
 
-        _logger.debug("{0} - ".format(sys._getframe().f_code.co_name))
+        _logger.debug("{0} - ".format(inspect.currentframe().f_code.co_name))
```

Comment:
I changed from sys._getframe() to inspect.currentframe() as the first one was also working but searching, it seemed that the proper way was to use inspect.currentframe()

Reply:
I prefer to use inspect.currentframe() as it is a standard way/code that I could use without the need to change a string manually.

Reply:
the sending process is going to be in execution for a while, currently 60 seconds, and that it will end and then restarted, do you think that memory consumption could be a problem ?

Reply:
if we have to change, should we do something like : 
frame = inspect.currentframe()
_logger.debug("{0} - ".format(frame.f_code.co_name))
del frame

Reply:
I will change all the occurrences  when we will consolidate the direction.